colors.css

```diff
@@ -39,13 +40,13 @@
   --border-grey: #e5e7eb;
   --border-input: #d0d0d0;
   --text-input: #afafaf;
-  --dark-blue: rgba(17,24,39,1);
+  --dark-blue: rgba(17, 24, 39, 1);
   --box-shadow: rgba(0, 0, 0, 0.3);
   color: var(--textgrey);
 }
 
 .muted {
-  color: var(--secondary-lighter);
+  color: var(--secondary-light);
```

Comment:
use different descriptive name, `muted` doesn't convey any meaning as such

Reply:
muted in design terms is understood as something of light tone in terms of text. So I feel the name is fine. But will definitely take your suggestion to use classes that are more easily understood